firefox_puppeteer/ui/windows.py

```diff
@@ -60,22 +60,17 @@ def close(self, handle):
         # TODO: Maybe needs to wait as handled via an observer
         return self.marionette.close_chrome_window()
 
-    def close_all(self, exceptions=None):
+    def close_all(self, exceptions=[]):
```

Comment:
Please never use other values than `None` as default for lists and other non-atomic types. This can result in unwanted behavior. So please leave the default value as it was before, and check for it inside the method first.